content/docs/reference/routes/from.mdx

```diff
@@ -113,21 +113,38 @@ routes:
     to: tcp://example.com:22
 ```
 
-:::tip **Note**
+### Wildcard processing behavior
 
-Wildcard From Routes (`*`) may take precedence over non-wildcard routes regardless of their ordering, and more specific wildcard routes will take precedence over less specific wildcard routes.
+Pomerium processes routes in the order they are defined in the configuration file. This holds true for external wildcard routes, too.
```

Comment:
I think the original note was intended to handle this case:

```yaml
routes:
  - from: https://*.example.com
    to: http://1.example.com
  - from: https://www.example.com
    to: http://2.example.com
```

It would appear that route (2) should never be matched as route (1) will always be matched first. However due to the way we setup routing, route (2) may actually get matched, even though it's ordered after route (1). 

Sorry for the confusion.

To put it another way:

> Pomerium processes routes in the order they are defined in the configuration file. However routes which do not contain wildcards (`*`) may take precedence over routes which do contain wildcards.

Reply:
Thanks for the clarification, and that makes sense to me. I'll update this to note accordingly.